### DISCUSSION TITLE
Build xDrip in Android Studio Electric Eel

### DISCUSSION THREAD

**Discussion by pinggi** [NightscoutFoundation/xDrip#2761] [General]
I cloned xDrip and opened it in Android Studio Electric Eel | 2022.1.1 Patch 2.
Run the build by clicking Build -> Make Project and got error about databinding

```
* What went wrong:
Execution failed for task ':app:dataBindingMergeDependencyArtifactsFastDebug'.
> Could not resolve all files for configuration ':app:fastDebugRuntimeClasspath'.
   > Could not resolve com.android.databinding:baseLibrary:2.3.0-dev.
     Required by:
         project :app > me.tatarka.bindingcollectionadapter2:bindingcollectionadapter:2.2.0 > com.android.databinding:library:1.3.1
         project :app > me.tatarka.bindingcollectionadapter2:bindingcollectionadapter:2.2.0 > com.android.databinding:adapters:1.3.1
      > Untested databinding version: com.android.databinding baseLibrary- 2.3.0-dev - 2.3.0-dev
   > Could not resolve com.android.databinding:library:1.3.
     Required by:
         project :app > me.tatarka.bindingcollectionadapter2:bindingcollectionadapter:2.2.0 > com.android.databinding:adapters:1.3.1
      > Untested databinding version: com.android.databinding library- 1.3 - 1.3
```

Is it possible to compile the project in Android Studio? Do I need to change build.gradle file somehow?

**Comment by fwolter** [comment]
If I remember correctly I also had this when I updated Gradle. Android Studio is asking you to do so after importing the project. You need to ignore this question. If this is the problem, starting over with a fresh Git clone should fix it.

**Comment by pinggi** [comment]
@fwolter That's true. I had errors that prevented the project from building, that is probably why I updated `Gradle`. I cloned the repo again and fixed the errors by installing `Java 1.8` and agreed to the licenses by running `sdkmanager --licenses`. Now I am able to build and run the project in the version of Android Studio.
